src/templates/careers.twig

```diff
@@ -1,9 +1,33 @@
-{% extends '_layouts/_generic_page_layout.twig' %}
+{% extends '_layouts/_base_page_layout.twig' %}
 
-{% block content %}
-  <section>
-    <h1>
-      Careers Page
-    </h1>
-  </section>
+{% block pageHeader %}
+  {% include '_partials/_page_header.twig' with {
+    title: 'Careers',
+    intro: '<p>“Be the change” and make a difference in the future of clean water. Join the EOSi team and discover how Empowering Clean Water® offers a fulfilling career today — and a better future for the world tomorrow.</p>'
+  } %}
+{% endblock %}
+
+{% block pageContent %}
+  <div>
+    {% for blogContent in [
+      {
+        date: 'Feb 20, 2022',
+        title: 'Position Title',
+        description: 'Maecenas faucibus mollis interdum. Aenean lacinia bibendum nulla sed consectetur. Donec ullamcorper nulla non metus auctor fringilla.',
+        btnText: 'Learn More'
+      },
+      {
+        date: 'Feb 20, 2022',
+        title: 'Position Title',
+        description: 'Maecenas faucibus mollis interdum. Aenean lacinia bibendum nulla sed consectetur. Donec ullamcorper nulla non metus auctor fringilla.',
+        btnText: 'Learn More'
+      }
+    ] %}
+      {% include '_partials/_blog_post.twig' with { blog: blogContent } %}
+    {% endfor %}
```

Comment:
Don't loop it here, just place includes in pages. 

Remember KISS (Keep It Simple Stupid)